boilerplate/bin/postInstall

```diff
@@ -16,7 +16,7 @@ if (["darwin", "linux"].includes(process.platform)) {
 
 // On iOS, make sure CocoaPods are installed
 if (process.platform === "darwin") {
-  run('if [ -d "ios" ]; then cd ios && pod install && cd -; fi')
+  run('npx pod-install')
```

Comment:
I kinda think the `if [ -d "ios" ]; then` is still useful here, in case someone goes full managed and removes their `ios` and `android` folders.

Reply:
This is OK, `pod-install` package handles that case:

![image](https://user-images.githubusercontent.com/374022/202061848-4c213efd-6494-41de-b86a-546a250cf732.png)

Reply:
It checks for darwin too, so we could simplify further

Reply:
![image](https://user-images.githubusercontent.com/374022/202065641-d6ccab33-dfb7-4ae8-a6d6-b472a10190fa.png)

Windows check for sanity